tests/unit/test_general_options.py

```diff
@@ -333,15 +336,23 @@ class       Command group for CIM classes.
       'test': 'innows'},
      None, OK],  # Only tests that the option is accepted
 
-
     ['Verify --warn general option.',
      {'general': ['-s', 'http://blah', '--warn'],
       'cmdgrp': 'connection',
       'args': ['show']},
      {'stdout': [''],
       'rc': 0,
       'test': 'innows'},
-     None, OK],
+     None, OK],  # Only tests that the option is accepted
+
+    ['Verify --warn general option.',
+     {'general': ['-s', 'http://blah', '--warn'],
+      'cmdgrp': 'connection',
+      'args': ['show']},
+     {'stderr': ["Pywbemcli support for Python", "is deprecated"],
+      'rc': 0,
+      'test': 'innows'},
+     None, PYTHON2],  # Only tests that the option is accepted
```

Comment:
This variant of the testcase really wants to test the deprecation message. That message is issued on Python 2.7 and 3.4, but the testcase is only run on Python 2.7. I think the `PYTHON2` variable should be changed to become `PYTHON_DEPRECATED` or so, and be True for both of these Python versions.